mockserver/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6
+FROM python
```

Comment:
can you force a version? Else it will fall back on python:latest which might break suddenly.